### ISSUE TITLE
hmset raises CommandError when passed an actual array

### ISSUE THREAD

**Issue by wdrury** [sds/mock_redis#134]
Where $redis is a MockRedis, this works:
```
$redis.hmset("woot", "boo", "bar")
```
... but this causes assert_has_args to raise an error:
```
$redis.hmset(["woot", "boo", "bar"])
```

The official redis/redis-rb allows me to do either.

**Comment by sds** [comment]
Thanks for the report, @wdrury. Would happily accept a pull request fixing this!

**Comment by JeffLuckett** [comment]
I have addressed this in this PR: https://github.com/sds/mock_redis/pull/173

**Comment by sds** [comment]
Fixed in #173.
